testplan/web_ui/testing/src/Report/reportUtils.js

```diff
@@ -216,7 +216,7 @@ const GetCenterPane = (
   selectedEntries
 ) => {
   const logs = state.logs || [];
-  const selectedDescription = selectedEntries.map((element) => {
+  const selectedDescription = selectedEntries.slice(-1).map((element) => {
```

Comment:
Can we display description of Testplan/Multitest/Testsuite in assertion panel if they are the bottom-most selection (no testcase selected yet)